soco/core.py

```diff
@@ -931,10 +935,14 @@ def switch_to_line_in(self):
         Raises SoCoException (or a subclass) upon errors.
 
         """
+        if not source:
```

Comment:
Since we are relying on the boolean value of a SoCo instance anyway, why not turn it around:
```python
if source:
    ...
else:
    ....
```